shanghai/main.py

```diff
@@ -14,48 +14,65 @@
 
 def exception_handler(loop, context):  # pylint: disable=unused-argument
     f = io.StringIO()
-    print("exception_handler context:", file=f)
+    print("=== Unhandled Exception ===", file=f)
+    print("- Context -", file=f)
     pprint(context, stream=f)
-    traceback.print_exc(file=f)
```

Comment:
See https://github.com/chireiden/shanghai/pull/30/commits/e87a27c01d66534f0560c9fa07a45f4ebee616c1. Never printed anything for me but `NoneType`. The traceback is instead printed using the respective `.print_stack` methods.